README.md

```diff
@@ -49,6 +49,7 @@ Other packages exists to integrate this SDK into the most common frameworks.
 
 - [Symfony](https://github.com/getsentry/sentry-symfony)
 - [Laravel](https://github.com/getsentry/sentry-laravel)
+- [Nette](https://github.com/Salamek/raven-nette)
```

Comment:
I would create a subsection for non-official integrations, so we could add also others like the Wordpress one or https://github.com/facile-it/sentry-module